### PR TITLE
updating DR

### PR DESCRIPTION
**Pull request by bernarddeluna**

### CODE REVIEW

File: docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md [hunk 1]
```diff
@@ -21,6 +21,55 @@ manually during a cross-region disaster:
...
+| **Cold** | You have no spare tire, so you must call someone to come to you with a new tire and replace it. Your trip stops until help arrives to make the repair. |
+| **Warm** | You have a spare tire and a replacement kit, so you can get back on the road using what you have in your car. However, you must stop your journey to repair the problem. |
+| **Hot**  | You have special run-flat tires. You might need to slow down a little, but there is no immediate impact on your journey. Your tires run well enough that you can continue (although you must eventually address the issue). |
+
+In DXP Cloud, we classify our DR strategy as *Warm*.
+
+It's not *Cold* since you don't have to contact our support in order to solve that problem for you. It's not *Hot* because you need human intervention to handle it. We consider it to be *Warm* since we provide you with all the tools to realize the procedure.
+
+
+
```
Comment (zenorocha) [inline docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md]: Remove extra line breaks

File: docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md [hunk 2]
```diff
@@ -21,6 +21,55 @@ manually during a cross-region disaster:
...
+| **1:00 pm**  | Our automatic backup is triggered, which will generate a copy of our Database and Document Library. |
+| **2:00 pm**  | Our automatic backup is triggered once again since we configured it to run every hour. |
+| **2:34 pm**  | We are notified that the entire *eu-west2* region has been compromised. This means we'll need to enable our DR environment, which is located in the *us-west1* region, in order to minimize downtime.<br/>From **2:00 pm** until **2:34 pm** there were no backups generated, therefore we'll need to restore the **2:00 pm** backup of the Production database and document library to the DR environment. In addition, we'll get the latest Build number from Production and deploy it to the DR environment, validate if the VPN has been configured properly in the DR environment, and finally change the Custom Domain to serve traffic to users.<br/>The amount of time those steps will take depends on the size of your application and how fast your team will be able to respond, but let's say it took 16 minutes. |
+| **3:00 pm**  | Everything is back to normal and now we're serving traffic through the DR environment. |
+
+
+
```
Comment (zenorocha) [inline docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md]: Remove extra line breaks

File: docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md [hunk 3]
```diff
@@ -21,6 +21,55 @@ manually during a cross-region disaster:
...
+###  **2. Once the disaster is no longer present**
+
+
```
Comment (zenorocha) [inline docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md]: Remove extra line break

File: docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md [hunk 4]
```diff
@@ -21,6 +21,55 @@ manually during a cross-region disaster:
...
+|              |                                                              |
+| ------------ | ------------------------------------------------------------ |
+| **4:00 pm**  | we are notified that the *eu-west2* region has been recovered and the incident is over, which means that we are ready to go back to our original Production environment |
+| **4:16 pm** | We'll perform a manual backup since there has been new user data generated during the period we were using the DR environment. |
+| **4:21 pm**  | We can use the manual backup we just generated to restore the Production environment and then change the Custom Domain. |
+
+Now we're back to serving traffic through the Production environment and our DR environment will stay running for when we need it again.
+
+
+
```
Comment (zenorocha) [inline docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md]: Remove extra line breaks

File: docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md [hunk 5]
```diff
@@ -159,3 +208,14 @@ the the Production environment.
 1. Remove the custom domain from the Production environment.
 1. Update the DNS records. For more information, see the [Custom Domain](https://help.liferay.com/hc/en-us/articles/360032856292) article.
 1. Click _Update Custom Domain_.
+
+
+
+
```
Comment (zenorocha) [inline docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md]: Remove extra line breaks

File: docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md [hunk 6]
```diff
@@ -21,6 +21,55 @@ manually during a cross-region disaster:
 * [During an Incident](#during-an-incident)
 * [Post-incident Recovery](#post-incident-recovery)
```
Comment (zenorocha) [inline docs/dxp-cloud/latest/en/troubleshooting/disaster-recovery.md]: Missing update of anchors.

**Comment by justinchoi001** [comment]
Reviewing. See https://github.com/jrhoun/liferay-learn/pull/110

**Comment by jrhoun** [comment]
Merged. Thank you.

**Pull request closed without merging**